test/join.jl

```diff
@@ -102,7 +102,7 @@ module TestJoin
     categorical!(dt1, :B)
     join(dt1, dt1, on = [:A, :B], kind = :inner)
 
-    # Test that Array{Nullable} works when combined with NullableArray (#1088)
```

Comment:
Could still say that this mixes nullable and non-nullable array (the old description looks incorrect?). Same below.